modules/local/benchmarking/location_plots/main.nf

```diff
@@ -0,0 +1,15 @@
+process LOCATION_PLOT {
+    label "process_single"
+
+    conda "bioconda::matplotlib=3.5.1 bioconda::seaborn=0.11.2"
+    container 'uphl/seaborn'
```

Comment:
Matplotlib is an internal dependency of seaborn, so you don't have to explicitly add it here. Because of this, you should be able to use the [biocontainer for seaborn](https://biocontainers.pro/tools/seaborn) here. For multi-tool containers, you can build your custom images using [seqera containers](https://seqera.io/containers/).